### PR TITLE
Fix tut

### PR DESCRIPTION
**Pull request by tieswesterborg**
Fixed some typos in the tutorial
Added definition of a risky object

### CODE REVIEW

File: src/components/pages/Tutorial.vue [hunk 1]
```diff
@@ -13,9 +13,10 @@
                 <div class="columnleft">
                     <h3>Find Task</h3>
                     <p>The goal of the Find task is to find and annotate high-risk objects.</p>
-                    <p>You are able to navigating through a bounded area in Google Street View. </p>
+                    <p>We define a high-risk object as an object that is touched by many different people (doorhandles, trash bins and park benches).</p>
```
Comment (arbreurkes) [inline src/components/pages/Tutorial.vue]: Slightly different wording maybe: "We define a high-risk object as an object that bight be touched by many different people..."

File: src/components/pages/Tutorial.vue [hunk 2]
```diff
@@ -38,7 +39,7 @@
                     <h3>Verify Task</h3>
                     <p>In this Verify task, the goal is to find annotated objects that are high-risk and to select the annotation that is most precise and clear in terms of location and snapshot.</p>
                     <p>You will be presented with the different snapshots proposed by fellow workers and a street view panorama where the context of the objects can be seen. </p>
-                    <p>You need to assesses whether an annotated object is high-risk and either select the best fitting snapshot or mark all object as `No’-Risk’. If you really don’t know, you should select the ‘UNCERTAIN’ button.</p>
+                    <p>You need to assesses whether an annotated object is high-risk and either select the best fitting snapshot or mark all object as `No Risk’. If you really don’t know, you should select the ‘UNCERTAIN’ button.</p>
```
Comment (arbreurkes) [inline src/components/pages/Tutorial.vue]: typo: "assess", ", and", "the object"

**Review by arbreurkes — CHANGES_REQUESTED** [review]
Just the small changes please :)

File: src/components/pages/Tutorial.vue (hunk 2 above)
Comment (tieswesterborg) [reply src/components/pages/Tutorial.vue]: Fixed :)

File: src/components/pages/Tutorial.vue [hunk 3]
```diff
@@ -13,7 +13,7 @@
                 <div class="columnleft">
                     <h3>Find Task</h3>
                     <p>The goal of the Find task is to find and annotate high-risk objects.</p>
-                    <p>We define a high-risk object as an object that is touched by many different people (doorhandles, trash bins and park benches).</p>
+                    <p>We define a high-risk object as an object that bight be touched by many different people (doorhandles, trash bins and park benches).</p>
```
Comment (arbreurkes) [inline src/components/pages/Tutorial.vue]: Typo: "bight"

**Review by arbreurkes — CHANGES_REQUESTED** [review]
One typo :')

**Review by arbreurkes — APPROVED** [review]

**Pull request merged**